test/features/resources.feature

```diff
@@ -0,0 +1,13 @@
+@resources
+Feature: Resource UI
+
+    Scenario Outline: Link resource should create a link to its URL
+        Given "Admin" as the persona
+        When I create a resource with name "<name>" and URL "<url>"
+        And I press the element with xpath "//a[contains(@title, '<name>') and contains(string(), '<name>')]"
+        Then I take a screenshot
+        And I should see "<url>"
+
+        Examples:
+        | name | url |
+        | link | http://www.qld.gov.au |
```

Comment:
can you pick another url, incapsula does not like automated tests sometimes.

Reply:
Will do. Merging to master first so that CircleCI recognises our config.